politeiad/backendv2/tstorebe/plugins/pi/hooks.go

```diff
@@ -140,6 +141,49 @@ func (p *piPlugin) proposalNameIsValid(name string) bool {
 	return p.proposalNameRegexp.MatchString(name)
 }
 
+// proposalStartDateIsValid returns whether the provided start date is valid.
+//
+// A valid start date of a proposal must be in the future.
+func (p *piPlugin) proposalStartDateIsValid(start int64) bool {
+	return start > time.Now().Unix()
```

Comment:
This needs some sort of Add. Require at least a day or something?

Reply:
We can add in a plugin setting for the minimum start date (seconds from current time).  I think it should default to 1 week though to account for the length of a proposal vote.

Reply:
This will also need to be added to the policy reply and the gui will need to use it for start date input.

Reply:
sweet will do